go/grpcweb/wrapper.go

```diff
@@ -9,88 +9,44 @@ import (
 	"io"
 	"net/http"
 	"strings"
-	"time"
 
 	"github.com/gorilla/websocket"
-	"github.com/rs/cors"
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/grpclog"
 )
 
-var (
-	internalRequestHeadersWhitelist = []string{
-		"U-A", // for gRPC-Web User Agent indicator.
```

Comment:
Somewhat confused by the inclusion of this header @MarcusLongmuir as the `U-A` Header does not appear anywhere in the grpc-web client codebase. Do you recall where this came from?